app/assets/stylesheets/master_detail.scss

```diff
@@ -0,0 +1,102 @@
+.master-detail-grids{
```

Comment:
I think we need to follow the same pattern for the css ie:
data_table.css:
```
 *= require dx.common
 *= require dx.light
 *= require master_detail.scss
```

Then in the relevant project:
application.css
`*= require data_table.css`